qemu/tests/bridge_vlan.py

```diff
@@ -243,6 +243,7 @@ def netperf_vlan(client='main_vm', server='localhost',
                               (vm_vlan_if[vm_index], vm.name), test.log.info)
         ping_vlan(vm, dest=host_vlan_ip, vlan_if=vm_vlan_if[vm_index],
                   session=session)
+        utils_net.restart_guest_network(sessions[vm_index])
```

Comment:
I think this should move before `ping_vlan `? And in the `for` loop, the session is defined in Line230 --> `session = sessions[vm_index]`

Reply:
Agree with you, changed it, please help me review it again, thanks a lot.